plugins/jade/src/test/scala/org/broadinstitute/monster/sbt/ClassGeneratorSpec.scala

```diff
@@ -354,6 +419,11 @@ class ClassGeneratorSpec extends AnyFlatSpec with Matchers with EitherValues {
        |      _root_.io.circe.derivation.renaming.snakeCase,
        |      _root_.scala.None
        |    )
+       |
+       |  def init(): StructColumn = {
+       |    StructColumn(
+       |      comment = _root_.scala.Option.empty[_root_.$structPackage.RowComment])
```

Comment:
Looks like a bug here, `$structPackage` not being substituted properly.

Reply:
Unless we're confusingly using the literal string "$structPackage" in the tests...

Reply:
I think it is substituting out the `$structPackage` value with "rab.oof" when it actually runs the tests. I just re-ran the test and had it print the actual output, which looks like this: 
```
case class StructColumn(
comment: _root_.scala.Option[_root_.rab.oof.RowComment])

object StructColumn {de / Test / executeTests 1s
  implicit val encoder: _root_.io.circe.Encoder[StructColumn] =
    _root_.io.circe.derivation.deriveEncoder(
      _root_.io.circe.derivation.renaming.snakeCase,
      _root_.scala.None
    )

  def init(): StructColumn = {
    StructColumn(
      comment = _root_.scala.Option.empty[_root_.rab.oof.RowComment])
  }
}
```

Reply:
Oh! I missed the `s` at the start of each string.

Reply:
They are defined above as:
```
  private val testPackage = "foo.bar"
  private val structPackage = testPackage.reverse
```